.gitmodules

```diff
@@ -1,5 +1,4 @@
-
 [submodule "owncloud-android-library"]
 	path = owncloud-android-library
-	url = git://github.com/owncloud/android-library.git
-	branch = develop
+	url = https://github.com/owncloudChalmers/android-library.git
```

Comment:
I'll change it to the proper one when merging. As mentioned in #661, we had to change this to be able to build because the Gradle support was not yet merged into your 'android-library' and we therefore could not build our 'android' using Gradle, but now that it is merged the problem is gone.